cluster-autoscaler/go.mod

```diff
@@ -22,6 +22,11 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
+	github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/as v1.0.335
```

Comment:
We don't accept any new dependencies for new cloud providers. Your cloud provider should have all the necessary stuff inside the cloud provider src directory. Please check other cloud providers for reference (example: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/alicloud)

Reply:
I have solved it.